Casks/e/es-de.rb

```diff
@@ -0,0 +1,48 @@
+cask "es-de" do
+  arch arm: "macOSApple", intel: "macOSIntel"
+
+  on_arm do
+    version "3.2.0,184126565"
+    sha256 "9893c0aa15947c7d2e29ab177c561d5f968056194e1ba280a024291ae3574ae1"
+  end
+  on_intel do
+    version "3.2.0,184126636"
+    sha256 "21b9659e833b8895f143de8e16f2fdda671c32fa2ea1185a7e9301741318c5fc"
+  end
+
+  url "https://gitlab.com/es-de/emulationstation-de/-/package_files/#{version.csv.second}/download",
+      verified: "gitlab.com/es-de/emulationstation-de/"
+  name "ES-DE"
+  name "EmulationStation Desktop Edition"
+  desc "Frontend for browsing and launching games from your multi-platform collection"
+  homepage "https://www.es-de.org/"
+
+  livecheck do
+    url "https://gitlab.com/api/v4/projects/18817634/repository/files/latest_release.json/raw?ref=master"
+    regex(%r{/package_files/(\d+)/download}i)
+    strategy :json do |json|
```

Comment:
```suggestion
    strategy :json do |json, regex|
```

@mvaude Just one small change that I overlooked in the `livecheck` block and we should be about good to go here.